services/apis/guppy/guppyTasks.js

```diff
@@ -0,0 +1,31 @@
+const guppyProps = require('./guppyProps.js');
+const user = require('../../../utils/user.js');
+const { Gen3Response, getCookie, getAccessTokenHeader } = require('../../../utils/apiUtil');
+const { Bash, takeLastLine } = require('../../../utils/bash');
+const { container } = require('codeceptjs');
+const bash = new Bash();
```

Comment:
some of these imports seems never been used

Reply:
ur right